Changelog.md

```diff
@@ -33,6 +33,7 @@ Breaking Changes:
  * General: Remove assembly instruction aliases ``sha3`` and ``suicide``
  * General: C99-style scoping rules are enforced now. This was already the case in the experimental 0.5.0 mode.
  * General: Disallow combining hex numbers with unit denominations (e.g. ``0x1e wei``). This was already the case in the experimental 0.5.0 mode.
+ * Interface: Remove "clone contract" feature.
```

Comment:
I think we should mention the name of the commandline option which is gone. Will do that myself.